mage_ai/data_cleaner/cleaning_rules/impute_values.py

```diff
@@ -157,57 +137,67 @@ class StringImputeSubRule(TypeImputeSubRule):
 
     def accepted_dtypes(self):
         return self.ACCEPTED_DTYPES
-    
+
     def evaluate(self, column):
         """
         Rule:
         1. If there are no null entries, no suggestion
         2. If the dataset was identified as timeseries, suggest sequential imputation
         3. Else, if more than MODE_PROP_LB of nonnull entries are a single value, use
-           imputation with mode 
+           imputation with mode
         4. Else, if less than RAND_EMPTY_UB ratio of entries are null, use random imputation
         5. Else suggest no imputation (no good fit)
         """
-        if self.statistics[f'{column}/null_value_rate']  == 0:
+        if self.statistics[f'{column}/null_value_rate'] == 0:
             return ImputationStrategy.NOOP
-        if (self.is_timeseries and 
-            self.statistics[f'{column}/max_null_seq']  <= self.MAX_NULL_SEQ_LENGTH and 
-            self.df[column].notna().iloc[0]):
+        if (self.is_timeseries and
+                self.statistics[f'{column}/max_null_seq'] <= self.MAX_NULL_SEQ_LENGTH and
+                self.df[column].notna().iloc[0]):
             return ImputationStrategy.SEQ
         elif self.statistics[f'{column}/mode_ratio'] >= self.MODE_PROP_LB:
             return ImputationStrategy.MODE
-        elif self.statistics[f'{column}/null_value_rate']  <= self.RAND_EMPTY_UB:
+        elif self.statistics[f'{column}/null_value_rate'] <= self.RAND_EMPTY_UB:
             return ImputationStrategy.RANDOM
-        return ImputationStrategy.NOOP
+        return ImputationStrategy.CONSTANT
 
 
 class ImputeValues(BaseRule):
-    RULESET = (
-        CategoricalImputeSubRule,
-        DateTimeImputeSubRule,
-        NumericalImputeSubRule,
-        StringImputeSubRule
-    )
+    RULESET = (CategoricalImputeSubRule, DateTimeImputeSubRule, NumericalImputeSubRule,
```

Comment:
I actually like the original style better. One item per line. In that way,  it'll be clearer to see the change every time you add an item.

Reply:
Agreed, I'll work on changing back the style.

Reply:
Changed the formatting back so that tuples, array, and arguments like above are now expanded for increased readability.